tasks/main.yml

```diff
@@ -14,6 +14,10 @@
   set_fact:
     cluster: "{{ 'openshift' if 'openshift' in oc_version.stdout else 'kubernetes' }}"
 
+- name: 'Include variables based on ansible version'
+  include_vars: ansible_26.yml
```

Comment:
Do you need to import the 25 variables as well?

Reply:
I just use them to override 25 variables.